bot/exts/evergreen/cheatsheet.py

```diff
@@ -0,0 +1,97 @@
+import random
+import re
+import typing as t
+from urllib.parse import quote_plus
+
+from discord import Embed
+from discord.ext import commands
+from discord.ext.commands import BucketType, Context
+
+from bot.constants import Channels, Colours, ERROR_REPLIES
+
+ERROR_MESSAGE = """
+Unknown cheat sheet. Please try to reformulate your query.
+
+**Examples**:
+```md
+.cht read json
```

Comment:
Could do: Get the command prefix from `constants.Client.prefix` rather than hard coding it. If you decide to do this, please do it throughout the file, rather than just this line.

This is purely optional, as I don't think there are plans to change lancebot's prefix on the python server any time soon. It only really impacts us as the dev team :)